modules/c++/re/source/RegexPCRE.cpp

```diff
@@ -20,39 +20,41 @@
  *
  */
 
-#if defined(USE_PCRE)
+#include "re/Regex.h" // this has to come before the #ifdef checks below
 
-#include "re/PCRE.h"
+#if !defined(__CODA_CPP11)
 
-re::PCRE::PCRE(const std::string& pattern, int flags) :
-    mPattern(pattern), mPCRE(NULL)
+#include <sstream>
+
+re::Regex::Regex(const std::string& pattern) :
+    mPattern(pattern), OVECCOUNT(999), mPCRE(NULL), mOvector(OVECCOUNT)
 {
     if (!mPattern.empty())
-        compile(mPattern, flags);
+    {
+        compile(mPattern);
+    }
 }
 
-void re::PCRE::destroy()
+void re::Regex::destroy()
 {
     if (mPCRE != NULL)
     {
         pcre_free(mPCRE);
     }
 }
 
-re::PCRE::~PCRE()
+re::Regex::~Regex()
 {
     destroy();
 }
 
-re::PCRE::PCRE(const re::PCRE& rhs)
+re::Regex::Regex(const re::Regex& rhs) :
+    mPattern(rhs.mPattern), OVECCOUNT(999), mPCRE(NULL), mOvector(OVECCOUNT)
```

Comment:
Don't want to get out of sync with having 999 in two spots if we can help it.  Should be able to set from `rhs.OVECCOUNT`) or if you ditch the variable altogether, size `mOvector` based on `rhs`'s size.